style.css

```diff
@@ -0,0 +1,191 @@
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 80%;
+  margin: auto auto 20px auto;
+  border: 3px solid black;
+}
+
+header li {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 8px;
+}
+
+header div:nth-child(2) ul li {
+  padding: 0 12px 0 0;
+}
+
+#listbooks li {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 8px;
+}
+
+li p {
+  margin: 0;
+}
+
+footer p {
+  padding: 1px 20px;
+}
+
+header div:nth-child(1) {
+  width: 40%;
+}
+
+header div:nth-child(1) p {
+  margin: 8px 20px;
+}
+
+header div:nth-child(1) p a {
+  text-decoration: none;
+  color: black;
+  font-size: 18px;
+}
+
+header div:nth-child(2) {
+  width: 60%;
+}
+
+header div:nth-child(2) nav {
+  margin: 10px 15px;
+}
+
+header div:nth-child(2) ul {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
```

Comment:
### OPTIONAL
- [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)